baseapp/baseapp.go

```diff
@@ -367,17 +364,6 @@ func (app *BaseApp) setIndexEvents(ie []string) {
 	}
 }
 
-// Router returns the legacy router of the BaseApp.
-func (app *BaseApp) Router() sdk.Router {
```

Comment:
We probably should have added an API-breaking changelog for this baseapp removal